Tasks/DownloadFileShareArtifactsV0/Strings/resources.resjson/ko-KR/resources.resjson

```diff
@@ -0,0 +1,39 @@
+{
+  "loc.friendlyName": "Download Build Artifacts",
```

Comment:
Looks like these were copied from "DownloadBuildArtifacts" and not removed. All the non en-US resource files should not be committed since these are not the right localized versions.

Reply:
done